app/src/main/java/com/example/roomies/NewCircleActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.example.roomies;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.ClipData;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
```

Comment:
Are you addressing the imports or the entire NewCircleActivity class? 

The imports are needed to bind to the layout elements in the view (text box, buttons etc.) 

The NewCircleActivity class is separate from CircleActivity and JoinCircleActivity so it will be needed:
1. CircleActivity - presents user the options of creating a new circle or joining an existing circle
2. NewCircleActivity - shows the join code of the newly created circle before proceeding to the home screen
3. JoinCircleActivity - allows user to join an existing circle using a join code
^^I will add the above to the comments of each class